packages/amplify-codegen-appsync-model-plugin/src/visitors/appsync-java-visitor.ts

```diff
@@ -753,6 +763,67 @@ export class AppSyncModelJavaVisitor<
     return ['SuppressWarnings("all")', ...annotations].filter(annotation => annotation);
   }
 
+  protected generateAuthRules(authDirectives: AuthDirective[]): string {
+    const rules: string[] = [];
+    authDirectives.forEach(directive => {
+      directive.arguments?.rules.forEach(rule => {
+        const authRule = [];
+        switch (rule.allow) {
+          case AuthStrategy.owner:
+            authRule.push('allow = AuthStrategy.OWNER');
+            if (rule.ownerField) {
+              authRule.push(`ownerField = "${rule.ownerField}"`);
+            }
+            if (rule.identityClaim) {
+              authRule.push(`identityClaim = "${rule.identityClaim}"`);
+            }
+            break;
+          case AuthStrategy.private:
+            authRule.push('allow = AuthStrategy.PRIVATE');
+            if (rule.ownerField) {
+              authRule.push(`ownerField = "${rule.ownerField}"`);
+            }
+            if (rule.identityClaim) {
+              authRule.push(`identityClaim = "${rule.identityClaim}"`);
+            }
+            break;
+          case AuthStrategy.public:
+            authRule.push('allow = AuthStrategy.PUBLIC');
```

Comment:
`ownerField` and `identityClaim` only apply to `AuthStrategy.OWNER`, so you can remove them from the `AuthStrategy.PRIVATE` and `AuthStrategy.PUBLIC` cases.

Reply:
Removed ownerField and identityClaim